### PR TITLE
Adding AJO Entity Schema

### PR DESCRIPTION
**Pull request by anshchauhan**
The initial requirement was to enrich feedback dataset with meaningful user friendly values instead of adobe internal ids like variantId, journeyVersionId, messagePublicationId, campaignVersionId as we provide today.

We are introducing an Adobe Journey Optimizer specific schema called AJO Entity Schema. This dataset based on this schema will act as an uber dataset for all the major AJO entities. Instead of dumping the whole entities with complex schemas into this dataset we'll denormalise the data such that clients can easily join feedback/tracking datasets to this dataset to get meaningful information like subject/title/messageName etc. Clients can export and join this with messageDeliveryFeedback schema to get the desired result.

The intention is to create schema with the following structure:
image

### CODE REVIEW

**Comment by anshchauhan** [comment]
@prabhum2 @cmathis Could you please review. I've added the use case in description.

CC: @sawhneyh @chetanmeh

File: extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+              "type": "string",
+              "description": "Is not frozen and might remain same between different versions of message publication."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/ajo/experiement": {
+          "title": "AJO Experiment Entity Specific Fields",
+          "type": "object",
+          "description": "AJO Experiment Entity Specific Fields",
+          "properties": {
```
Comment (jainaks) [inline extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json:103]: Don't we need experimentId here?
Is treatmentId an UUID?

File: extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+              "title": "Journey Version ID",
+              "type": "string",
+              "description": "Represents a frozen version of a Journey"
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
```
Comment (mjasrotia) [inline extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json:126]: Please add the local definition "ajo" here too

File: extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json (hunk 1 above)
Comment (mjasrotia) [reply extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json:103]: Please have this approved from one of your ajo SMEs once all your internal discussions resolved. Also, please leave a comment on this PR once its ready to be reviewed.

**Comment by anshchauhan** [comment]
@mjasrotia the comments has been addressed. I see the PR is still failing, but I don't exactly understand the lint reason. Could you please help?

**Review by jainaks — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Comment by prabhum2** [comment]
@anshchauhan build is failing due to lint issue . Please ping me on slack if you need any help fixing build issue.

**Comment by anshchauhan** [comment]
@mjasrotia @prabhum2 Thank you for your help. The build seems to fail at `incompatibility-check` step which does not seem to show errors related to my change. Anything I'm missing?

**Comment by anshchauhan** [comment]
Got it. Just needed to rebase. @prabhum2 @mjasrotia could you please merge it? Thanks.

**Pull request merged**